teuthology/run.py

```diff
@@ -161,9 +161,12 @@ def validate_tasks(config):
         # return the default value for tasks
         return []
 
+    msg = 'Expecting (bullet) list of tasks instead of unordered map'
```

Comment:
Maybe print out config['tasks'] here as well?  Maybe: msg = "Expected a list of tasks and instead got: {0}".format(config['tasks'])